lightning/src/ln/channelmanager.rs

```diff
@@ -461,6 +461,24 @@ pub struct ChannelManager<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref,
 	logger: L,
 }
 
+/// Chain-related parameters used to construct a new `ChannelManager`.
+///
+/// Typically, the block-specific parameters are derived from the best block hash for the network,
+/// as a newly constructed `ChannelManager` will not have created any channels yet. These parameters
```

Comment:
Does it make sense to note that once you instantiate the ChannelManager, you have to start connecting blocks starting from the ones provided?

Reply:
Yeah, though added to `ChannelManager::new` docs since that is where `block_connected` is discussed and `ChainParameters` is used.